src/actions/actions.ts

```diff
@@ -5131,6 +5132,34 @@ class MoveFormat extends BaseMovement {
   }
 }
 
+// Used for gUU
```

Comment:
It'd probably be good to move these next to the other weirdo motions I made for dd yy cc etc.